src/test/java/models/testUser.java

```diff
@@ -30,9 +30,11 @@ public void testUserHand() {
     public void testTakeCard() {
         //Arrange
         User u = new User();
+        Deck deck = new Deck();
 
         //Act
-        u.takeCard();
+        deck.build();
+        u.takeCard(deck.deal());
 
```

Comment:
This should be zero. TakeCard in the User class does nothing. It doesn't add any card to the hand.